### PR TITLE
support for chrome.webRequest

### PR DESCRIPTION
**Pull request by dylanowen**
This adds initial support for https://developer.chrome.com/extensions/webRequest#type-BlockingResponse. I deviated from the Event and EventSource pattern used elsewhere but due to the listener function's parameters I wasn't sure what else to do. Let me know if you have any suggestions.

### CODE REVIEW

File: bindings/src/main/scala/chrome/webRequest/BlockingRequestEventSource.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    new SubscriptionImpl(event, callback)
+  }
+}
+
+
+object WebRequestEventSource {
+  implicit def eventAsEventSource[T <: js.Function](event: WebRequestEvent[T]): WebRequestEventSource[T] =
+    new WebRequestEventSourceImpl(event)
+}
```
Comment (lucidd) [inline bindings/src/main/scala/chrome/webRequest/BlockingRequestEventSource.scala]: This and a few other files are missing a new line at the end of the file.

**Review by lucidd — CHANGES_REQUESTED** [review]
Hey sorry for the long delay. Thanks you very much for this contribution 👍 The PR looks good there are just some new lines missing at the end of some files. Will merge after this is fixed.

**Comment by dylanowen** [comment]
Thanks for the review, I believe I cleaned up all the missing newlines. Let me know if there is anything else.

**Review by lucidd — APPROVED** [review]

**Comment by lucidd** [comment]
Looks fine now 👍  Thanks again for this great PR!

**Pull request merged**